### ISSUE TITLE
How do I install avconv on windows?

### ISSUE THREAD

**Issue by hdubb** [rg3/youtube-dl#7158]
I installed ffmpeg but I want to try avconv.

My windows is 64 bit so what I did was:
https://libav.org/download/ and went all the way down to Windows nightly and Release Builds, clicked on 64 bit windows builds, release-gpl/, then downloaded libav-11.3-win64.7z

I then extracted libav-11.3-win64.7z and moved the folder usr to my C drive and renamed it avconv

Then I went into my environmental variables and changed the path to c:\avconv\bin

These steps are similar to how I installed ffmpeg but it is still not working.

**Comment by jaimeMF** [comment]
If running `avconv.exe`from the command line doesn't work, there's nothing that youtube-dl can do. You should ask support to the avconv developers.

**Comment by 5j9** [comment]
I used the binary files from https://github.com/BtbN/FFmpeg-Builds/releases and extracted them into a folder that was in my system path. See [On Windows, how should I set up ffmpeg and youtube-dl? Where should I put the exe files?](https://github.com/ytdl-org/youtube-dl#on-windows-how-should-i-set-up-ffmpeg-and-youtube-dl-where-should-i-put-the-exe-files) for more info on modifying PATH variable.

(I know this is an old closed issue, but it showed up as the first result when I was googling for an answer and I thought adding this comment might help someone else)
